pom.xml

```diff
@@ -380,6 +380,12 @@ SOFTWARE.
             <skip>false</skip>
           </configuration>
         </plugin>
+        <!--
+         @todo #789:30min Enable forbidden-apis plugin.
+          Check code for compliance with code style described
+          in CONTRIBUTING.md. Fix as many classes as time allows
+          and than update this todo to fix the rest.
+        -->
```

Comment:
@andreoss please don't create new tasks as a todos, if it's not a task completion.

Reply:
@g4s8 Removed it.